--- lcms2-2.26/include/lcms2.h	2014-03-17 18:09:30.000000000 +0200
+++ lcms2-2.26/include/lcms2.h-new	2014-06-24 23:57:20.499345988 +0300
@@ -31,7 +31,10 @@
 // ********** Configuration toggles ****************************************
 
 // Uncomment this one if you are using big endian machines
-// #define CMS_USE_BIG_ENDIAN   1
+#include <endian.h>
+#if __BYTE_ORDER == __BIG_ENDIAN
+#define USE_BIG_ENDIAN   1
+#endif
 
 // Uncomment this one if your compiler/machine does NOT support the
 // "long long" type.
