--- SPECS/stm-target-openssl.spec	2014-04-09 22:05:07.000000000 +0300
+++ SPECS/stm-target-openssl.spec-new	2014-04-22 21:50:17.004961721 +0300
@@ -4,6 +4,7 @@
 Release: 30
 Source0: ftp://ftp.openssl.org/source/openssl-%{version}.tar.gz
 Patch0: openssl-0.9.8g-hwrandom.patch
+Patch1: openssl.patch
 
 License: Freely distributable
 Group: System Environment/Libraries
@@ -98,6 +99,7 @@
 %prep
 %setup -q -n openssl-%{version}
 %patch0 -p1
+%patch1 -p1
 
 %build
 %target_setup
