--- a/tools/depmod.c	2015-01-13 20:57:28.464245724 +0200
+++ b/tools/depmod.c	2015-01-13 20:59:52.916962039 +0200
@@ -1229,6 +1229,7 @@
 	return 0;
 }
 
+#if 0
 static void depmod_modules_sort(struct depmod *depmod)
 {
 	char order_file[PATH_MAX], line[PATH_MAX];
@@ -1281,6 +1282,7 @@
 corrupted:
 	fclose(fp);
 }
+#endif
 
 static int depmod_symbol_add(struct depmod *depmod, const char *name,
 					bool prefix_skipped, uint64_t crc,
@@ -1955,6 +1957,7 @@
 
 static int output_builtin_bin(struct depmod *depmod, FILE *out)
 {
+#if 0
 	FILE *in;
 	struct index_node *idx;
 	char infile[PATH_MAX], line[PATH_MAX], modname[PATH_MAX];
@@ -1989,6 +1992,7 @@
 	index_write(idx, out);
 	index_destroy(idx);
 	fclose(in);
+#endif
 
 	return 0;
 }
@@ -2583,7 +2587,7 @@
 		goto cmdline_modules_failed;
 	}
 
-	depmod_modules_sort(&depmod);
+	//depmod_modules_sort(&depmod);
 	err = depmod_load(&depmod);
 	if (err < 0)
 		goto cmdline_modules_failed;
